packages/template-retail-react-app/app/components/experience/image-tile/index.jsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {AspectRatio, Image} from '@chakra-ui/react'
+
+/**
+ * This is a simple Image Tile component that can be used inside any Layout component
+ *
+ * @param imageProps
+ * @param ratio
+ * @returns {JSX.Element}
+ */
+const ImageTile = ({imageProps, ratio = 1, ...rest}) => {
+    return (
+        <AspectRatio ratio={ratio}>
+            <figure className={'photo-tile-figure'}>
+                <picture>
+                    <source srcSet={imageProps.src?.tablet} media="(min-width: 48em)" />
+                    <source srcSet={imageProps.src?.desktop} media="(min-width: 64em)" />
+                    <Image
+                        className={'photo-tile-image image-fluid'}
+                        src={imageProps.src?.mobile ? imageProps.src?.mobile : imageProps.src}
+                        ignoreFallback={true}
+                        alt={imageProps.alt}
+                        title={imageProps.title}
+                        {...rest}
+                    />
+                </picture>
+            </figure>
+        </AspectRatio>
+    )
+}
+
+ImageTile.propTypes = {
+    imageProps: PropTypes.object,
+    ratio: PropTypes.number,
+    caption: PropTypes.string
```

Comment:
Where did these props come from? Below is what the API returns for a `ImageTile` component:

```
            "data": {
              "image": {
                "_type": "Image",
                "focalPoint": {
                  "_type": "Imagefocalpoint",
                  "x": 0.5,
                  "y": 0.5
                },
                "metaData": {
                  "_type": "Imagemetadata",
                  "height": 179,
                  "width": 410
                },
                "url": "https://development-functional38-qa222.demandware.net/on/demandware.static/-/Library-Sites-RefArchSharedLibrary/default/dwd35a6963/placeholder 410x179.4.png"
              }
            },
```

So ideally we implement those props and nothing else.

Reply:
Currently my implementation of the base components will spread the serialized component onto the provided component. 

So technically the only prop you'd have now is `id`, `data` and `typeId`. I'm wondering if this is something we should change to make the component implementations look nicer and more reusable. 

Lets talk about that tomorrow.